core/trie/trie_test.go

```diff
@@ -12,6 +12,12 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func hexToFelt(hex string) *felt.Felt {
```

Comment:
There are multiple of this in our codebase already. Can you move the existing one to a `testutils` package or something and use that everywhere please.